utils/openzwave/Makefile

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (C) 2011 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=openzwave
+PKG_REV:=164
+PKG_VERSION:=1.4.$(PKG_REV)
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+#PKG_SOURCE_PROTO:=git
+#PKG_SOURCE_URL:=https://github.com/OpenZWave/open-zwave.git
+#PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)
+#PKG_MD5SUM:=1c8c1c7ed92daa9e464316dd62237188
+PKG_SOURCE_URL:=http://old.openzwave.com/downloads
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)
+PKG_SOURCE_VERSION:=$(PKG_REV)
+PKG_USE_MIPS16:=0 
```

Comment:
Yes, flag PKG_USE_MIPS16:=0 is required. Without this flag packages build failed.